setup.py

```diff
@@ -100,7 +100,7 @@ def read_version_py(infname):
     #   `pip install tahoe-lafs[sftp]` would not install requirements
     #   specified by Twisted[conch].  Since this would be the *whole point* of
     #   an sftp extra in Tahoe-LAFS, there is no point in having one.
-    "Twisted[tls,conch] >= 18.4.0",
+    "Twisted[tls,conch] >= 18.4.0, < 20.0.0",
 
```

Comment:
There should probably be a comment about why the limitation here (20.3.0 *does* support Python2.7, although it also says it's the last release that will).

Reply:
If CI configuration indeed is the cause of failing tests (and not having an upper bound), I guess I will have to drop this change.  Holding off for now.

Reply:
Dropped the upper bound in 81a88bf.  I guess the CI will remain broke because those changes haven't been merged yet.